### ISSUE TITLE
Keep old migrators around but disabled

### ISSUE THREAD

**Issue by mariusvniekerk** [regro/cf-scripts#704]
Rather than simply making them no longer exist, it might be a good idea to have them live on for a while so that immigrated repos can still add the bot-rerun label.
Pull requests for repositories not affected by the bot will not be made.

**Comment by CJ-Wright** [comment]
Maybe we need to make an `archived` migration folder in the pinning repo?

**Comment by CJ-Wright** [comment]
Ahhh, for the case of pinning migrations this should be just a rerender PR.

**Comment by mariusvniekerk** [comment]
Yeah, but some kind of model where things still behave as expected would be nice :)

**Comment by CJ-Wright** [comment]
done

**Comment by CJ-Wright** [comment]
Sorry, not done this is for old migration yamls
